builder/src/lib.rs

```diff
@@ -190,6 +190,11 @@ impl Default for ImageOptions {
 }
 
 pub fn build_and_sign_image(fmc: &FwId, app: &FwId, opts: ImageOptions) -> anyhow::Result<Vec<u8>> {
+    let image = build_image(fmc, app, opts)?;
+    get_image_bytes(&image)
+}
+
+pub fn build_image(fmc: &FwId, app: &FwId, opts: ImageOptions) -> anyhow::Result<ImageBundle> {
```

Comment:
This is also signing the image, it's just returning the whole image struct instead of just the bytes.

Seems like build_and_sign_image can just return the ImageBundle and then callers can call `to_bytes` as needed

Reply:
Updated